src/DynamoCore/Core/Threading/DelegateBasedAsyncTask.cs

```diff
@@ -23,7 +23,7 @@ internal DelegateBasedAsyncTask(IScheduler scheduler)
         {
         }
         
-        internal DelegateBasedAsyncTask(IScheduler scheduler, Action action)
+        public DelegateBasedAsyncTask(IScheduler scheduler, Action action)
```

Comment:
`DelegateBasedAsyncTask` is used in DynamoWebServer. so if we want DynamoCore know nothing about DynamoWebServer we can't use `[assembly: InternalsVisibleTo("DynamoWebServer")]` anymore